doc-content/enterprise-only/admin-and-config/managing-business-central-data-sources-con.adoc

```diff
@@ -1,12 +1,11 @@
 [id='managing-business-central-data-sources-con']
+= Data source management
 
-= Managing data sources
-
-{CENTRAL} provides data source management which gives you the ability of defining data sources for accessing a database. These data sources are then used by other {CENTRAL} components such as the data sets. A database driver, on the other hand, is used for enabling communication between a data source and the targeted database.
+Business Central provides data source management features that enable you to define data sources for accessing a database. These data sources are then used by other Business Central components such as the data sets. A database driver enables communication between a data source and the targeted database.
```

Comment:
Why is Business Central not {central} here?

Reply:
Good catch!